zerver/lib/onboarding.py

```diff
@@ -131,6 +134,35 @@ def send_initial_direct_message(user: UserProfile) -> None:
         disable_external_notifications=True,
     )
 
+    custom_welcome_bot_message_string = ""
+    custom_welcome_bot_message_text = ""
```

Comment:
I think, `_string` and `_text` suffix don't create much distinction & are confusing to understand how they differ.

`welcome_bot_custom_test_message_content` (or simply `welcome_bot_test_message_content`) & `welcome_bot_custom_message_content` sounds better?

Reply:
I'll update the variable names accordingly.